theme/templates/accounts/profile.html

```diff
@@ -157,15 +157,6 @@ <h4 class="text-muted">
                                 </div>
 
                                 <div class="row">
-                                    {# Username #}
-                                    <fieldset class="col-sm-4">
```

Comment:
@pkdash Thanks for pointing it out. Have fixed this issue so user profile form validation now passes without username as a required field.